--- xfce4-screenshooter-1.7.9/src/xfce4-screenshooter.desktop.in.in~	2009-10-31 00:53:18.000000000 +0200
+++ xfce4-screenshooter-1.7.9/src/xfce4-screenshooter.desktop.in.in	2012-04-14 12:20:36.734894116 +0300
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Version=1.0
-_Name=Screenshot
+_Name=Screenshooter
 _Comment=Take screenshots of the entire screen, of the active window or of a region
 Icon=applets-screenshooter
 Exec=xfce4-screenshooter
