--- xfce4-screenshooter-1.8.1/Makefile.am~	2012-05-02 20:20:11.000000000 +0200
+++ xfce4-screenshooter-1.8.1/Makefile.am	2012-05-17 20:47:58.765551200 +0200
@@ -208,7 +208,7 @@
 
 
 # C documentation
-cdocdir = $(datadir)/doc/xfce4-screenshooter/html/C/
+cdocdir = $(datadir)/xfce4/doc/C/
 cdoc_DATA = docs/manual/C/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -217,7 +217,7 @@
 endif
 endif
 
-cimagesdir = $(datadir)/doc/xfce4-screenshooter/html/C/images
+cimagesdir = $(datadir)/xfce4/doc/C/images
 cimages_DATA = \
 	docs/manual/C/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/C/images/xfce4-screenshooter-dialog2.png \
@@ -226,7 +226,7 @@
 
 
 # Catalan documentation
-cadocdir = $(datadir)/doc/xfce4-screenshooter/html/ca
+cadocdir = $(datadir)/xfce4/doc/ca
 cadoc_DATA = docs/manual/ca/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -235,7 +235,7 @@
 endif
 endif
 
-caimagesdir = $(datadir)/doc/xfce4-screenshooter/html/ca/images
+caimagesdir = $(datadir)/xfce4/doc/ca/images
 caimages_DATA = \
 	docs/manual/ca/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/ca/images/xfce4-screenshooter-dialog2.png \
@@ -244,7 +244,7 @@
 
 
 # Danish documentation
-dadocdir = $(datadir)/doc/xfce4-screenshooter/html/da
+dadocdir = $(datadir)/xfce4/doc/da
 dadoc_DATA = docs/manual/da/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -253,7 +253,7 @@
 endif
 endif
 
-daimagesdir = $(datadir)/doc/xfce4-screenshooter/html/da/images
+daimagesdir = $(datadir)/xfce4/doc/da/images
 daimages_DATA = \
 	docs/manual/da/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/da/images/xfce4-screenshooter-dialog2.png \
@@ -261,7 +261,7 @@
 	docs/manual/da/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Portuguese documentation
-ptdocdir = $(datadir)/doc/xfce4-screenshooter/html/pt
+ptdocdir = $(datadir)/xfce4/doc/pt
 ptdoc_DATA = docs/manual/pt/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -270,7 +270,7 @@
 endif
 endif
 
-ptimagesdir = $(datadir)/doc/xfce4-screenshooter/html/pt/images
+ptimagesdir = $(datadir)/xfce4/doc/pt/images
 ptimages_DATA = \
 	docs/manual/pt/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/pt/images/xfce4-screenshooter-dialog2.png \
@@ -278,7 +278,7 @@
 	docs/manual/pt/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Spanish documentation
-esdocdir = $(datadir)/doc/xfce4-screenshooter/html/es
+esdocdir = $(datadir)/xfce4/doc/es
 esdoc_DATA = docs/manual/es/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -287,7 +287,7 @@
 endif
 endif
 
-esimagesdir = $(datadir)/doc/xfce4-screenshooter/html/es/images
+esimagesdir = $(datadir)/xfce4/doc/es/images
 esimages_DATA = \
 	docs/manual/es/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/es/images/xfce4-screenshooter-dialog2.png \
@@ -295,7 +295,7 @@
 	docs/manual/es/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # French documentation
-frdocdir = $(datadir)/doc/xfce4-screenshooter/html/fr
+frdocdir = $(datadir)/xfce4/doc/fr
 frdoc_DATA = docs/manual/fr/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -304,7 +304,7 @@
 endif
 endif
 
-frimagesdir = $(datadir)/doc/xfce4-screenshooter/html/fr/images
+frimagesdir = $(datadir)/xfce4/doc/fr/images
 frimages_DATA = \
 	docs/manual/fr/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/fr/images/xfce4-screenshooter-dialog2.png \
@@ -312,7 +312,7 @@
 	docs/manual/fr/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Galician documentation
-gldocdir = $(datadir)/doc/xfce4-screenshooter/html/gl
+gldocdir = $(datadir)/xfce4/doc/gl
 gldoc_DATA = docs/manual/gl/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -321,7 +321,7 @@
 endif
 endif
 
-glimagesdir = $(datadir)/doc/xfce4-screenshooter/html/gl/images
+glimagesdir = $(datadir)/xfce4/doc/gl/images
 glimages_DATA = \
 	docs/manual/gl/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/gl/images/xfce4-screenshooter-dialog2.png \
@@ -329,7 +329,7 @@
 	docs/manual/gl/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Indonesian documentation
-iddocdir = $(datadir)/doc/xfce4-screenshooter/html/id
+iddocdir = $(datadir)/xfce4/doc/id
 iddoc_DATA = docs/manual/id/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -338,7 +338,7 @@
 endif
 endif
 
-idimagesdir = $(datadir)/doc/xfce4-screenshooter/html/id/images
+idimagesdir = $(datadir)/xfce4/doc/id/images
 idimages_DATA = \
 	docs/manual/id/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/id/images/xfce4-screenshooter-dialog2.png \
@@ -346,7 +346,7 @@
 	docs/manual/id/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Italian documentation
-itdocdir = $(datadir)/doc/xfce4-screenshooter/html/it
+itdocdir = $(datadir)/xfce4/doc/it
 itdoc_DATA = docs/manual/it/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -355,7 +355,7 @@
 endif
 endif
 
-itimagesdir = $(datadir)/doc/xfce4-screenshooter/html/it/images
+itimagesdir = $(datadir)/xfce4/doc/it/images
 itimages_DATA = \
 	docs/manual/it/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/it/images/xfce4-screenshooter-dialog2.png \
@@ -363,7 +363,7 @@
 	docs/manual/it/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Japanese documentation
-jadocdir = $(datadir)/doc/xfce4-screenshooter/html/ja
+jadocdir = $(datadir)/xfce4/doc/ja
 jadoc_DATA = docs/manual/ja/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -372,7 +372,7 @@
 endif
 endif
 
-jaimagesdir = $(datadir)/doc/xfce4-screenshooter/html/ja/images
+jaimagesdir = $(datadir)/xfce4/doc/ja/images
 jaimages_DATA = \
 	docs/manual/ja/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/ja/images/xfce4-screenshooter-dialog2.png \
@@ -380,7 +380,7 @@
 	docs/manual/ja/images/xfce4-screenshooter-zimagez-result-dialog.png
 
 # Turkish documentation
-trdocdir = $(datadir)/doc/xfce4-screenshooter/html/tr
+trdocdir = $(datadir)/xfce4/doc/tr
 trdoc_DATA = docs/manual/tr/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -389,7 +389,7 @@
 endif
 endif
 
-trimagesdir = $(datadir)/doc/xfce4-screenshooter/html/tr/images
+trimagesdir = $(datadir)/xfce4/doc/tr/images
 trimages_DATA = \
 	docs/manual/tr/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/tr/images/xfce4-screenshooter-dialog2.png \
@@ -398,7 +398,7 @@
 
 
 # Siplified Chinese documentation
-zhcndocdir = $(datadir)/doc/xfce4-screenshooter/html/zh_CN
+zhcndocdir = $(datadir)/xfce4/doc/zh_CN
 zhcndoc_DATA = docs/manual/zh_CN/index.html
 if MAINTAINER_MODE
 if ENABLE_XSLTPROC
@@ -407,7 +407,7 @@
 endif
 endif
 
-zhcnimagesdir = $(datadir)/doc/xfce4-screenshooter/html/zh_CN/images
+zhcnimagesdir = $(datadir)/xfce4/doc/zh_CN/images
 zhcnimages_DATA = \
 	docs/manual/zh_CN/images/xfce4-screenshooter-dialog1.png \
 	docs/manual/zh_CN/images/xfce4-screenshooter-dialog2.png \
