commit 504a9935f3fa900636ae5f8f40e8182b4fc293ec
Author: Jérôme Guelfucci <jeromeg@xfce.org>
Date:   Tue Dec 7 14:41:30 2010 +0100

    Replace libxfcegui4 by libxfce4ui (bug #6438).

diff --git a/Makefile.am b/Makefile.am
index 55b06b6..e1e12ed 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -37,7 +37,7 @@ lib_libscreenshooter_la_CFLAGS = \
   @GTK_CFLAGS@ \
 	@GLIB_CFLAGS@ \
 	@LIBXFCE4UTIL_CFLAGS@ \
-	@LIBXFCEGUI4_CFLAGS@ \
+	@LIBXFCE4UI_CFLAGS@ \
 	@SOUP_CFLAGS@ \
   -DPACKAGE_LOCALE_DIR=\"$(localedir)\"
 
@@ -45,7 +45,7 @@ lib_libscreenshooter_la_LIBADD = \
 	@EXO_LIBS@ \
   @GTK_LIBS@ \
 	@LIBXFCE4UTIL_LIBS@ \
-	@LIBXFCEGUI4_LIBS@ \
+	@LIBXFCE4UI_LIBS@ \
   @GLIB_LIBS@ \
 	@SOUP_LIBS@
 
@@ -78,7 +78,7 @@ src_xfce4_screenshooter_CFLAGS = \
 	@GTK_CFLAGS@ \
 	@GLIB_CFLAGS@ \
 	@LIBXFCE4UTIL_CFLAGS@ \
-	@LIBXFCEGUI4_CFLAGS@ \
+	@LIBXFCE4UI_CFLAGS@ \
 	@GTHREAD_CFLAGS@ \
 	@SOUP_CFLAGS@ \
 	-DPACKAGE_LOCALE_DIR=\"$(localedir)\"
@@ -90,7 +90,7 @@ src_xfce4_screenshooter_LDFLAGS = \
 	@GTHREAD_LIBS@ \
 	@SOUP_LIBS@ \
 	@LIBXFCE4UTIL_LIBS@ \
-	@LIBXFCEGUI4_LIBS@
+	@LIBXFCE4UI_LIBS@
 
 src_xfce4_screenshooter_LDADD = lib/libscreenshooter.la
 	
@@ -119,6 +119,7 @@ panel_plugin_xfce4_screenshooter_plugin_CFLAGS =	\
 	-I$(top_srcdir)/lib/	\
 	@EXO_CFLAGS@ \
 	@LIBXFCE4PANEL_CFLAGS@ \
+	@LIBXFCE4UI_CFLAGS@ \
 	@GTHREAD_CFLAGS@ \
 	@SOUP_CFLAGS@
 
@@ -128,7 +128,8 @@ panel_plugin_xfce4_screenshooter_plugin_LDFLAGS = \
 panel_plugin_xfce4_screenshooter_plugin_LDFLAGS = \
 	@LIBXFCE4PANEL_LIBS@ \
 	@GTHREAD_LIBS@ \
-	@SOUP_LIBS@
+	@SOUP_LIBS@ \
+	@LIBXFCE4UI_LIBS@
 
 panel_plugin_xfce4_screenshooter_plugin_LDADD = lib/libscreenshooter.la
 
diff --git a/configure.ac b/configure.ac
index 84bffae..3b0e44c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -48,7 +48,7 @@ dnl *** Check for required packages ***
 dnl ***********************************
 XDT_CHECK_PACKAGE([LIBXFCE4PANEL], [libxfce4panel-1.0], [4.4.0])
 XDT_CHECK_PACKAGE([LIBXFCE4UTIL], [libxfce4util-1.0], [4.4.0])
-XDT_CHECK_PACKAGE([LIBXFCEGUI4], [libxfcegui4-1.0], [4.4.0])
+XDT_CHECK_PACKAGE([LIBXFCE4UI], [libxfce4ui-1], [4.7.0])
 XDT_CHECK_PACKAGE([GTHREAD], [gthread-2.0], [2.16.0])
 XDT_CHECK_PACKAGE([GTK], [gtk+-2.0], [2.16.0])
 XDT_CHECK_PACKAGE([GLIB], [glib-2.0], [2.16.0])
commit 785a32403d5b653ca914f5ce52e76a9de7e5b915
Author: Jérôme Guelfucci <jeromeg@xfce.org>
Date:   Tue Dec 7 14:50:57 2010 +0100

    Missing bits...

diff --git a/lib/screenshooter-dialogs.h b/lib/screenshooter-dialogs.h
index b97b5fa..33121a3 100644
--- a/lib/screenshooter-dialogs.h
+++ b/lib/screenshooter-dialogs.h
@@ -33,7 +33,7 @@
 
 #include <gtk/gtk.h>
 #include <libxfce4util/libxfce4util.h>
-#include <libxfcegui4/libxfcegui4.h>
+#include <libxfce4ui/libxfce4ui.h>
 
 
 GtkWidget *screenshooter_actions_dialog_new (ScreenshotData *sd);
diff --git a/lib/screenshooter-utils.h b/lib/screenshooter-utils.h
index f40929a..d1b4fd2 100644
--- a/lib/screenshooter-utils.h
+++ b/lib/screenshooter-utils.h
@@ -32,7 +32,7 @@
 #include <time.h>
 
 #include <libxfce4util/libxfce4util.h>
-#include <libxfcegui4/libxfcegui4.h>
+#include <libxfce4ui/libxfce4ui.h>
 
 
 
diff --git a/panel-plugin/screenshooter-plugin.c b/panel-plugin/screenshooter-plugin.c
index 2d69ac5..60ae066 100644
--- a/panel-plugin/screenshooter-plugin.c
+++ b/panel-plugin/screenshooter-plugin.c
@@ -28,7 +28,7 @@ t */
 #include <gtk/gtk.h>
 #include <gdk/gdkx.h>
 
-#include <libxfcegui4/libxfcegui4.h>
+#include <libxfce4ui/libxfce4ui.h>
 #include <libxfce4panel/xfce-panel-plugin.h>
 #include <libxfce4panel/xfce-panel-convenience.h>
 
commit e8ab483698caceeae0e436dc7695c068dbfdfa5a
Author: Jérôme Guelfucci <jeromeg@xfce.org>
Date:   Thu Nov 25 22:46:39 2010 +0100

    Use GtkIconTheme functions instead of xfce_themed_icon.
    
    This fixes the build with Xfce's git master branch and works the same.

diff --git a/panel-plugin/screenshooter-plugin.c b/panel-plugin/screenshooter-plugin.c
index b0ac320..2d69ac5 100644
--- a/panel-plugin/screenshooter-plugin.c
+++ b/panel-plugin/screenshooter-plugin.c
@@ -119,7 +119,11 @@ cb_set_size (XfcePanelPlugin *plugin, int size, PluginData *pd)
                                     pd->button->style->ythickness);
 
   TRACE ("Get the icon from the theme");
-  pb = xfce_themed_icon_load (SCREENSHOT_ICON_NAME, width);
+  pb = gtk_icon_theme_load_icon (gtk_icon_theme_get_default (),
+                                 SCREENSHOT_ICON_NAME,
+                                 width,
+                                 GTK_ICON_LOOKUP_FORCE_SIZE,
+                                 NULL);
 
   TRACE ("Set the new icon");
   gtk_image_set_from_pixbuf (GTK_IMAGE (pd->image), pb);
